hdijupyterutils/requirements.txt

```diff
@@ -1,8 +1,8 @@
 nose
 mock
 ipywidgets>5.0.0
-ipython==4.0.2
-ipykernel==4.2.2
+ipython>=4.0.2,<7
+ipykernel>=4.2.2,<5
 pandas==0.17.1
```

Comment:
With pandas being tied to this specific version, requirements files are in conflict. Can you update here? I don't remember why we pinned it like that in the past though...

Reply:
If you update here, you also need to udpate the setup.py file